### PR TITLE
style changes in devices mobile

### PR DESCRIPTION
**Pull request by karinisantanna**
Hi!

I made some small style changes  in devices mobile!
I hope it is useful! :)

### CODE REVIEW

File: styles/config.css [hunk 1]
```diff
@@ -342,3 +348,9 @@ footer li {
 footer a:hover {
   text-decoration: none;
 }
+
+@media only screen and (max-width: 768px) {
+ .social-links ul {
+    width: 325px;
```
Comment (cTxplorer) [inline styles/config.css:354]: @karinisantanna we should center align the social buttons here. Mind making necessary changes?

File: styles/config.css [hunk 2]
```diff
@@ -189,12 +189,18 @@ body {
 
   .main-content {
     margin-left: 0;
-    padding-left: 16px;
-    padding-right: 16px;
+    padding-left: 0px;
+    padding-right: 0px;
+    width: 90%;
```
Comment (cTxplorer) [inline styles/config.css:194]: @karinisantanna just curious why give 90% width over constant side padding? Did you find any issue in the original setup?

**Review by cTxplorer — CHANGES_REQUESTED** [review]

File: styles/config.css (hunk 1 above)
Comment (karinisantanna) [reply styles/config.css]: @cTxplorer Hi! yes, i revert this part! Can you validate please?

File: styles/config.css (hunk 2 above)
Comment (karinisantanna) [reply styles/config.css]: @cTxplorer I had the impression that the paddings were breaking the block, so i thought about reducing the width and using margin 0 auto to center it. 
But I checked it again and I think it was a problem with the mobile emulator I was using!

I revert this part! Can you validate please?

**Comment by karinisantanna** [comment]
Hi @cTxplorer ! I've already made the requested adjustments! Can you validate please?

**Pull request closed without merging**